lib/athena-queries.js

```diff
@@ -0,0 +1,49 @@
+import config from 'config';
+const { fetchesTable } = config.get('athena');
+
+module.exports = {
+  locationsMetadata: `
```

Comment:
Does this mean we can get rid of the existing locations query? I think yes right? 

https://github.com/openaq/openaq-api/blob/develop/api/controllers/webhooks.js#L87

Actually, I think this webhook endpoint would crash currently if called since `locations.js` not longer exists.

Reply:
Didn't noticed that webhooks would broke. I'll review that part so we can have locations Athena sync logic is use there, as propose above.